File: Python developer/step07/7.01/7.1.05.py
'''

На вход вашей программе поступает список целых чисел через пробел.

Удалите из массива все элементы, значения которых являются нечетными.

Ввод:

Список целых чисел через пробел

Вывод:

Четные элементы массива

Sample Input 1:

1 2 3 4 5

Sample Output 1:

2 4

Sample Input 2:

0 0 0 0

Sample Output 2:

0 0 0 0

Sample Input 3:

1 1 1

Sample Output 3:

Sample Input 4:

0

Sample Output 4:

0

Sample Input 5:

1

Sample Output 5:

'''

n = [int(i) for i in input().split() if int(i)%2 == 0]
print(*n)

File: Diving into JavaScript: For Beginners/step07/7.01/7.1.05.js
/*

Перед вами та же программа, но теперь формат вывода сообщения изменен. Напишите функцию с одной инструкцией console.log(), 
в которой должны быть использованы спецсимволы.

Sample Input 1:

Футболка
200
3

Sample Output 1:

Вы выбрали "Футболка" по цене 200 рублей за штуку.
Количество: 3 шт.
Итого: 600 рублей.

Sample Input 2:

Платье
1500
2

Sample Output 2:

Вы выбрали "Платье" по цене 1500 рублей за штуку.
Количество: 2 шт.
Итого: 3000 рублей.

*/

const itemName = prompt();
const itemPrice = Number(prompt());
const quantity = Number(prompt());

function calculateTotal(n,p,q) {
    return `Вы выбрали \"${n}\" по цене ${p} рублей за штуку.\nКоличество: ${q} шт.\nИтого: ${q*p} рублей.`;
};

const message = calculateTotal(itemName, itemPrice, quantity);
console.log(message);
